cfn.yaml

```diff
@@ -20,6 +20,16 @@ Parameters:
         Description: Bucket where RiffRaff uploads artifacts on deploy
         Type: String
         Default: targeting-dist
+    TLSCert:
+        Type: String
+        Description: ARN of TLS certificate in US-EAST-1
+        Default: "arn:aws:acm:us-east-1:477621165360:certificate/781a1702-ca2e-4142-8acf-7ce23f52d106"
```

Comment:
I wonder if a default here makes sense? I think I'd be tempted to omit this.

Reply:
I'm unsure. Is doesn't seem like a security issue to have in Git, and is handy to have around so that it's faster to rebuild the infrastructure if needed